ghost/core/core/server/models/user.js

```diff
@@ -488,6 +488,32 @@ User = ghostBookshelf.Model.extend({
         return query.fetch(options);
     },
 
+    /**
+     * Returns users who should receive a specific type of alert
+     * @param {'free-signup'|'paid-started'|'paid-canceled'} type The type of alert to fetch users for
+     * @param {any} options
+     * @return {Promise<[Object]>} Array of users
+     */
+    getEmailAlertUsers(type, options) {
+        options = options || {};
+
+        let filter = 'status:active';
+        if (type === 'free-signup') {
+            filter += '+free_member_signup_notification:true';
+        } else if (type === 'paid-started') {
+            filter += '+paid_subscription_started_notification:true';
+        } else if (type === 'paid-canceled') {
+            filter += '+paid_subscription_canceled_notification:true';
+        }
+        return this.findAll(_.merge({filter, withRelated: ['roles']}, options)).then((users) => {
```

Comment:
This reads that the `filter` option can be overridden, which would make this method behave unusually - was this purposeful?

Reply:
> was this purposeful?

It was not, that's a good point. I think it should be other way around like below
```
_.merge(options, {filter, withRelated: ['roles']})
```

Reply:
Yeah, perhaps even `merge({}, options, {filter, withRelated: ['roles']});` so that we don't augment the options object passed in